packages/gcf-utils/test/gcf-utils.ts

```diff
@@ -212,4 +214,156 @@ describe('GCFBootstrapper', () => {
       sinon.assert.calledOnce(configStub);
     });
   });
+
+  describe('getProbotConfig', () => {
+    let bootstrapper: GCFBootstrapper;
+    let secretClientStub: v1.SecretManagerServiceClient;
+    let secretsStub: sinon.SinonStub;
+    let secretVersionNameStub: sinon.SinonStub;
+
+    beforeEach(() => {
+      secretClientStub = new v1.SecretManagerServiceClient();
+      bootstrapper = new GCFBootstrapper(secretClientStub);
+
+      secretVersionNameStub = sinon
+        .stub(bootstrapper, 'getLatestSecretVersionName')
+        .callsFake(() => {
+          return 'foobar';
+        });
+    });
+
+    afterEach(() => {
+      secretsStub.reset();
+      secretVersionNameStub.reset();
+    });
+
+    it('gets the config', async () => {
+      secretsStub = sinon
+        .stub(secretClientStub, 'accessSecretVersion')
+        .callsFake(() => {
+          return Promise.resolve([
+            {
+              payload: {
+                data: JSON.stringify({
+                  id: 1234,
+                  secret: 'foo',
+                  webhookPath: 'bar',
+                }),
+              },
+            },
+          ]);
+        });
+      await bootstrapper.getProbotConfig();
+      sinon.assert.calledOnce(secretsStub);
+      sinon.assert.calledOnceWithExactly(secretsStub, {name: 'foobar'});
+      sinon.assert.calledOnce(secretVersionNameStub);
+    });
+
+    it('throws on empty data', async () => {
+      secretsStub = sinon
+        .stub(secretClientStub, 'accessSecretVersion')
+        .callsFake(() => {
+          return Promise.resolve([
+            {
+              payload: {
+                data: '',
+              },
+            },
+          ]);
+        });
+
+      const configSpy = sinon.spy(bootstrapper, 'getProbotConfig');
+      try {
+        await configSpy();
+      } catch (e) {
+        // pass
+      }
+      expect(configSpy.threw());
+    });
+
+    it('throws on empty payload', async () => {
+      secretsStub = sinon
+        .stub(secretClientStub, 'accessSecretVersion')
+        .callsFake(() => {
+          return Promise.resolve([
+            {
+              payload: {},
+            },
+          ]);
+        });
+
+      const configSpy = sinon.spy(bootstrapper, 'getProbotConfig');
```

Comment:
I think this would be simpler if you did something like:
```js
assert.rejects(bootstrapper.getProbotConfig());
```

If I'm understanding the intent here right

Reply:
It is! 👍